Userland/Applications/PixelPaint/Tools/MoveTool.cpp

```diff
@@ -46,8 +48,16 @@ void MoveTool::on_mousemove(Layer* layer, MouseEvent& event)
     if (!layer)
         return;
 
+    auto sensitivity = 20;
+    auto updated_is_in_lower_right_corner = Gfx::IntRect(Gfx::IntPoint(layer->location().x() + layer->size().width() - sensitivity / 2, layer->location().y() + layer->size().height() - sensitivity / 2), Gfx::IntSize(sensitivity, sensitivity))
```

Comment:
For readability I'd split this huge thing into multiple locals.
Consider building a `Gfx::IntRect grab_rect` and `Gfx::IntPoint grab_rect_position` separately.